tests/generate_source/bn.src

```diff
@@ -0,0 +1 @@
+#error This file should not be included in compilation
```

Comment:
Can you add end-of-file newlines to `an.implicit.src`, `bn.src` and `cn.src` please? (Github's showing that these files do not have them)

Reply:
ok